test/draw/DrawStateTest.ooc

```diff
@@ -0,0 +1,54 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+use unit
+use draw
+use geometry
+
+DrawStateTest: class extends Fixture {
+	compareTransform: static func (resultTransform, correctTransform: FloatTransform3D, precision: Float) {
+		expect(resultTransform a, is equal to(correctTransform a) within(precision))
+		expect(resultTransform b, is equal to(correctTransform b) within(precision))
+		expect(resultTransform c, is equal to(correctTransform c) within(precision))
+		expect(resultTransform d, is equal to(correctTransform d) within(precision))
+		expect(resultTransform e, is equal to(correctTransform e) within(precision))
+		expect(resultTransform f, is equal to(correctTransform f) within(precision))
+		expect(resultTransform g, is equal to(correctTransform g) within(precision))
+		expect(resultTransform h, is equal to(correctTransform h) within(precision))
+		expect(resultTransform i, is equal to(correctTransform i) within(precision))
+		expect(resultTransform j, is equal to(correctTransform j) within(precision))
+		expect(resultTransform k, is equal to(correctTransform k) within(precision))
+		expect(resultTransform l, is equal to(correctTransform l) within(precision))
```

Comment:
Actually maybe this should be a function in `FloatTransform3D`. Not necessarily in this PR.